apps/ledger-live-desktop/src/renderer/components/DeviceAction/utils.ts

```diff
@@ -11,3 +12,18 @@ export function isDeviceNotOnboardedError(e: unknown) {
       (e.message.includes("0x6d06") || e.message.includes("0x6d07")))
   );
 }
+
+export function getNoSuchAppProviderLearnMoreMetadataPerApp(appName: string): {
+  learnMoreTextKey?: string;
+  learnMoreLink?: string;
+} {
+  switch (appName) {
+    case "Ledger Sync":
+      return {
+        learnMoreLink: urls.learnMoreLedgerSync,
+        learnMoreTextKey: "errors.NoSuchAppOnProvider.learnMoreCTA",
+      };
+    default:
+      return {};
```

Comment:
we don't provide anything for default?

Reply:
@mcayuelas-ledger  we don't as learnMoreLink depends on apps context

Reply:
for example ACRE there is no learnMoreLink, if they want to add it, they should implement it here.

Reply:
Okey :)